dotCMS/src/main/java/com/dotmarketing/filters/TimeMachineFilter.java

```diff
@@ -245,7 +245,7 @@ private void sendFile(final File file, ServletRequest request, final ServletResp
 		}
 		resp.setContentType(mimeType);
 		resp.setContentLength((int) file.length());
-		try (FileInputStream fis = new FileInputStream(file)) {
+		try (FileInputStream fis = new FileInputStream(file)) { // todo: should use nio here
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)